ivw_integration.install

```diff
@@ -20,7 +21,7 @@ function ivw_integration_requirements($phase) {
     if (empty($config->get('site'))) {
       $requirements['ivw_integration'] = [
         'title' => t('IVW'),
-        'description' => t('IVW has not been configured yet. Please configure its settings from the <a href="@url">IVW settings page</a>.', ['@url' => \Drupal::url('ivw_integration.admin_settings')]),
+        'description' => t('IVW has not been configured yet. Please configure its settings from the <a href="@url">IVW settings page</a>.', ['@url' => Url::fromRoute('ivw_integration.admin_settings')->getInternalPath()]),
```

Comment:
This doesn't route to the correct page. I think we need `Url::fromRoute('ivw_integration.admin_settings')->toString()` here

Reply:
Maybe this `Link::fromTextAndUrl('Conflict', Url::fromUri('https://www.drupal.org/project/conflict'))->toString()`